README.md

```diff
@@ -19,6 +20,8 @@ at http://ckanhostname/pod/validate.
 Installation
 ------------
 
+To install using a docker development environment, refer [here](ckan-docker/README.md)
```

Comment:
Let's pull this inline instead of having it in a separate document. The docker workflow is the primary way to setup a development environment.